doozerlib/cli/release_gen_assembly.py

```diff
@@ -46,9 +46,10 @@ def releases_gen_assembly(ctx, name):
 @click.option("--suggestions-url", metavar='SUGGESTIONS_URL', required=False,
               default="https://raw.githubusercontent.com/openshift/cincinnati-graph-data/master/build-suggestions/",
               help="When using --auto-previous, set custom suggestions URL, load from {major}-{minor}-{arch}.yaml")
+@click.option('--output-file', required=False, help='Specify a file path to write the doozer output to')
```

Comment:
"to write the generated assembly definition to" instead of doozer output

Reply:
Also we can add a shortflag `-o`

Reply:
Done!